guides/common/modules/con_overview-of-hosts-in-project.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Overview-of-Hosts-in-{Project}_{context}"]
+= Overview of Hosts in {Project}
+
+A host is any Linux client that {ProjectName} manages.
```

Comment:
Depends on the build target. orcharhino and upstream support Windows hosts AFAIK.